certified-connectors/AzureDigitalTwins/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1002 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AzureDigitalTwins",
+    "description": "Azure Digital Twins",
+    "version": "1.0"
+  },
+  "host": "digitaltwins.azure.net",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/digitaltwins/{twinid}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "result": {
+                  "type": "string",
+                  "description": "Results From the Twin.",
+                  "title": "Result"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Twin by Id",
+        "description": "Pass in the twin id to retrieve its details.",
```

Comment:
@sankumrav please make sure that all titles, summaries, descriptions are grammatically correct and are free of typographical errors. More information can be found in [English strings recommendations](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).